### PR TITLE
Add Befonts to the font category

### LINKED ISSUE #943: [Add] Befonts to the fonts category
### Site URL

https://befonts.com/

### Description of the above provided site

Befonts is a website that offers a wide variety of free and premium fonts for designers. With over 50,000 fonts to choose from, Befonts has something for everyone. Whether you're looking for a classic serif font or a modern sans serif font, Befonts has you covered.

### Category

Frontend

### Sub Category

fonts

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by AhorIsaac**
## Fixes Issue
Closes #943 

## Changes proposed

Add Befonts to the font category

### CODE REVIEW

**Comment by rupali-codes** [comment]
Please add _closes #issuenumber_ in the description of this PR as the issue is not referenced properly

**Comment by AhorIsaac** [comment]
I added **closes** #issuenumber in the description of the pull request.

**Comment by rupali-codes** [comment]
> I added **closes** #issuenumber in the description of the pull request.

Thanks, could you please make the suggested changes?

File: database/frontend/fonts.json [hunk 1]
```diff
@@ -39,5 +40,12 @@
         "url": "https://fontesk.com/",
         "category": "frontend",
         "subcategory": "fonts"
+    },
+    {
+        "name": "Befonts",
+        "description": "Befonts is a platform when you can download the best fonts, free fonts for personal or commercial use.",
```
Comment (CBID2) [inline database/frontend/fonts.json:46]: [Proofreading]
```suggestion
        "description": "Befonts is a platform where you download the best fonts, free fonts for personal or commercial use.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @AhorIsaac! I agree with @rupali-codes’ suggestion.

**Comment by AhorIsaac** [comment]
I did precisely as you instructed!

**Review by rupali-codes — CHANGES_REQUESTED** [review]
could you please undo your changes in .gitignore file as its not the part of your issue?

**Comment by rupali-codes** [comment]
Also, please resolve the merge conflicts

**Comment by AhorIsaac** [comment]
Done

**Review by ujjawaltyagii — APPROVED** [review]
lgtm

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**